metis-common/metis-common-utils/src/test/java/eu/europeana/metis/utils/CompressedFileHandlerGzTest.java

```diff
@@ -1,16 +1,16 @@
 package eu.europeana.metis.utils;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Path;
 import java.util.Collection;
 import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.Test;
 
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
my suggestion is to configure intelliJ to use metis-framework coding style (.../metis-framework/.idea/codeStyles/Project.xml), 
in metis-framework we avoid `.*` imports as much as possible please include only this ones:
```
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertNotNull;
```
and run format code command.

Reply:
Done